docs/deploy/aws.md

```diff
@@ -162,6 +162,21 @@ You need to request a certificate in the AWS Certificate Manager by following th
 the ARN of the new certificate. You also need to replace host-name with the hostname of choice like
 demo.datahubproject.io.
 
+To have the metadata [authentication service](https://datahubproject.io/docs/introducing-metadata-service-authentication/#configuring-metadata-service-authentication) enable and use [API tokens](https://datahubproject.io/docs/introducing-metadata-service-authentication/#generating-personal-access-tokens) from the UI you will need to set the configuration in the values.yaml for the `gms` and the `frontend` deployments. For example, using extraEnvs:
```

Comment:
Hey! Checkout this section of the values.yaml https://github.com/acryldata/datahub-helm/blob/master/charts/datahub/values.yaml#L163 If this section is enabled, you should see the authentication enabled. (It internally sets the below envs)

Reply:
Thanks for reviewing @dexter-mh-lee. Indeed the idea here is to make this auth process transparent from the main guide since in my case had to debug these connections in the EKS deployments to figure it out. I have made the modification you are suggesting, please let me know if it works.